doc/releases/index.rst

```diff
@@ -1,14 +1,52 @@
 .. _zephyr_release_notes:
 
-Release Notes
-#############
+Releases
+########
 
 Zephyr project is provided as source code and build scripts for different
 target architectures and configurations, and not as a binary image. Updated
-versions of the Zephyr project are released approximately every three-months.
+versions of the Zephyr project are released approximately every four months.
 
 All Zephyr project source code is maintained in a `GitHub repository`_.
 
+Release Life Cycle and Maintenance
+**********************************
+
+Periodic Releases
+=================
+
+The Zephyr project provides periodic releases every 4 months leading to the
+long term support releases approx. every 2 years. Priodic and non-LTS releases
+are maintained with updates, bug fixes and security related updates for at
+least two cycles, meaning that the project supports the most recent two
+releases in addition to the most recent LTS.
```

Comment:
Non-blocking: adding an example would be useful IMO

"For example, if the latest release is Zephyr vN.3 for some number N, then both vN.2 and vN.3 will continue to receive updates. However, vN.1 has reached end of life and will no longer receive updates unless it is an LTS."

Reply:
This is very complicated and difficult to read and understand IMO, KISS.